docs/5g-on-helium/mining-5g.mdx

```diff
@@ -0,0 +1,162 @@
+|  28  | 01/16/2023 |      1.28       | 5G POC Start |     1.00     |   34.01    |
+|  29  | 01/23/2023 |      1.28       | 5G POC Start |     1.00     |   35.29    |
+|  30  | 01/30/2023 |      1.28       | 5G POC Start |     1.00     |   36.58    |
+|  31  | 02/06/2023 |      1.28       | 5G POC Start |     1.00     |   37.86    |
+|  32  | 02/13/2023 |      1.28       | 5G POC Start |     1.00     |   39.14    |
+|  33  | 02/20/2023 |      1.28       | 5G POC Start |     1.00     |   40.43    |
+|  34  | 02/27/2023 |      1.28       | 5G POC Start |     1.00     |   41.71    |
+|  35  | 03/06/2023 |      1.28       | 5G POC Start |     1.00     |   42.99    |
+|  36  | 03/13/2023 |      1.28       | 5G POC Start |     1.00     |   44.28    |
+|  37  | 03/20/2023 |      1.28       | 5G POC Start |     1.00     |   45.56    |
+|  38  | 03/27/2023 |      1.28       | 5G POC Start |     1.00     |   46.84    |
+|  39  | 04/03/2023 |      1.28       | 5G POC Start |     1.00     |   48.13    |
+|  40  | 04/10/2023 |      1.28       | 5G POC Start |     1.00     |   49.41    |
+|  41  | 04/17/2023 |      1.28       | 5G POC Start |     1.00     |   50.69    |
+|  42  | 04/24/2023 |      1.28       | 5G POC Start |     1.00     |   51.97    |
+|  43  | 05/01/2023 |      1.28       | 5G POC Start |     1.00     |   53.26    |
+|  44  | 05/08/2023 |      1.28       | 5G POC Start |     1.00     |   54.54    |
+|  45  | 05/15/2023 |      1.28       | 5G POC Start |     1.00     |   55.82    |
+|  46  | 05/22/2023 |      1.28       | 5G POC Start |     1.00     |   57.11    |
+|  47  | 05/29/2023 |      1.28       | 5G POC Start |     1.00     |   58.39    |
+|  48  | 06/05/2023 |      1.28       | 5G POC Start |     1.00     |   59.67    |
+|  49  | 06/12/2023 |      1.28       | 5G POC Start |     1.00     |   60.96    |
+|  50  | 06/19/2023 |      1.28       | 5G POC Start |     1.00     |   62.24    |
+|  51  | 06/26/2023 |      1.28       | 5G POC Start |     1.00     |   63.52    |
+|  52  | 07/03/2023 |      1.28       | 5G POC Start |     1.00     |   64.81    |
+|  53  | 07/10/2023 |      1.28       | 5G POC Start |     1.00     |   66.09    |
+
+## How Do Hotspots Earn Helium MOBILE Tokens?
+
+The Helium blockchain rewards 5G Hotspots for providing 5G wireless coverage and verifying the
+Helium Network. 5G Hotspots are rewarded in the Helium MOBILE Token.
+
+During the Genesis Period, 5G Hotspots will earn MOBILE by reporting their status via a “heartbeat”,
+an automatic process of reporting to the Blockchain that the Small Cell Radio is online and
+registered with the SAS.
+
+| Reward Type | Chain Variable | Description                                                             |
+| :---------- | :------------- | :---------------------------------------------------------------------- |
+| Heartbeat   | `UNKNOWN`      | Rewarded to any 5G Hotspot that maintains a valid connection to the SAS |
```

Comment:
There are three chainvars, let's drop this table for now.

Reply:
agree on dropping. but maybe say below that we will start with "heartbeats" as a metric.

Reply:
dropping the larger emissions table or just the chain var table?